### ISSUE TITLE
Create a cron scheduled job that tests producing release binaries

### ISSUE THREAD

**Issue by cburgdorf** [argotorg/fe#761]
### What is wrong?

Every once in a while when we want to cut a release we are surprised to find out that cutting the release failed because there's some issue with either solc, gcc, boost or whatever. It would be nice if we had a job that runs once a day and produces actual binaries for all supported platforms. That way we can find out early instead of having to deal with it at release day.

### How can it be fixed

Basically do what `expensive.yams` does (but scheduling it once a day should be enough) and combine it with what our release job does except that it should not actually upload the binaries to the release page. But testing the binaries (e.g. compiling a contract) would be a nice extra gem.
